src/code/z_play.c

```diff
@@ -541,11 +561,15 @@ void Play_Update(PlayState* this) {
                         }
                     }
 
+#if OOT_DEBUG
                     if (!R_TRANS_DBG_ENABLED) {
                         Play_SetupTransition(this, this->transitionType);
                     } else {
                         Play_SetupTransition(this, R_TRANS_DBG_TYPE);
                     }
+#else
+                    Play_SetupTransition(this, this->transitionType);
+#endif
```

Comment:
This particular block was picky and didn't want to accept `if (!OOT_DEBUG || !R_TRANS_DBG_ENABLED) {`, a better match is welcome

Reply:
does it match with the debug check second, we've seen a couple cases of that already

Reply:
It doesn't unfortunately. I've only seen the debug check second work for `&& OOT_DEBUG`, has it also worked for `|| !OOT_DEBUG`?

Reply:
There's an `|| !OOT_DEBUG` that's like that in game.c currently (there's two, but one of them matches the other way too, and I don't remember which)